app/src/main/res/layout/time_dialog.xml

```diff
@@ -64,6 +65,45 @@
                 android:layout_height="wrap_content"
                 android:id="@+id/disableSleeptimerButton"/>
 
+        <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal">
+
+            <Button
+                android:id="@+id/extendSleepFiveMinutesButton"
```

Comment:
Could you please indent those with more spaces, so that the file uses consistent indention? I know it's different in different files but we should at least try to keep one scheme per file.

Reply:
I've indented it with 8 spaces now.